app/src/main/java/com/cpr3663/cpr_scouting_app/PostMatch.java

```diff
@@ -203,10 +202,10 @@ public void onClick(View view) {
 
                 // Log all of the data from this page
                 Globals.EventLogger.LogData(Constants.LOGKEY_DID_LEAVE_START, String.valueOf(postMatchBinding.checkboxDidLeave.isChecked()));
-//                Globals.EventLogger.LogData(Constants.LOGKEY_CLIMB_POSITION, postMatchBinding.spinnerClimbPosition.getSelectedItem().toString());
-//                Globals.EventLogger.LogData(Constants.LOGKEY_TRAP, postMatchBinding.spinnerTrap.getSelectedItem().toString());
-                // TODO : need to know how to build a multi-selected list of IDs (delimiter will be ":")
-//                Globals.EventLogger.LogData(Constants.LOGKEY_COMMENTS, postMatchBinding.dropComments.toString());
+                Globals.EventLogger.LogData(Constants.LOGKEY_CLIMB_POSITION, postMatchBinding.spinnerClimbPosition.getSelectedItem().toString());
+                Globals.EventLogger.LogData(Constants.LOGKEY_TRAP, postMatchBinding.spinnerTrap.getSelectedItem().toString());
+                // TODO need to know how to build a multi-selected list of IDs (delimiter will be ":")
+                Globals.EventLogger.LogData(Constants.LOGKEY_COMMENTS, postMatchBinding.dropComments.toString());
```

Comment:
This isn't right.  We don't want to log the comment strings.  We need to convert this to a colon separated list of comment ID's.  Also, remove the TODO comment above.

Reply:
Is that necessary for tonight?

Reply:
yes.  we have to have the logging files working so we can collect the files we generate tonight to work on Tableau flows and stuff next week.

Reply:
OK, I'm working on it.

Reply:
Spinners also need to log the ID of the selected item, not the string.